source/except.d.ts

```diff
@@ -29,8 +29,19 @@ type Filtered = Filter<'bar', 'foo'>;
 */
 type Filter<KeyType, ExcludeType> = IsEqual<KeyType, ExcludeType> extends true ? never : (KeyType extends ExcludeType ? never : KeyType);
 
+type ExceptOptions = {
+	/**
+	Disallow assigning non-specified properties.
+
+	Setting this to `false` is not recommended. Included for backwards-compatability.
+
+	@default false
+	*/
+	requireExactProps?: boolean;
+};
+
 /**
-Create a type from an object type without certain keys.
+Create a type from an object type without certain keys. Use option `requireExactProps: true` to disallow assigning additional properties to this type. Note that any omitted properties in the resulting type will be present in suggestions as `undefined`.
```

Comment:
This text should be in the docs for `ExceptOptions` instead. The only text we need here is just:

> We recommend setting the `requireExactProps` option to `true`.